sdk/servicebus/azure-messaging-servicebus/src/main/java/module-info.java

```diff
@@ -9,12 +9,8 @@
     exports com.azure.messaging.servicebus.administration.models;
     exports com.azure.messaging.servicebus.models;
 
-    opens com.azure.messaging.servicebus;
-    opens com.azure.messaging.servicebus.administration;
-    opens com.azure.messaging.servicebus.administration.models;
-    opens com.azure.messaging.servicebus.implementation;
-    opens com.azure.messaging.servicebus.implementation.models;
-    opens com.azure.messaging.servicebus.models;
+    opens com.azure.messaging.servicebus.administration.models to com.fasterxml.jackson.databind;
+    opens com.azure.messaging.servicebus.implementation.models to com.fasterxml.jackson.databind;
```

Comment:
Needed for test `ServiceBusManagementSerializerTest.deserializeNamespace`
accessible: module com.azure.messaging.servicebus does not "exports com.azure.messaging.servicebus.implementation.models" to module com.fasterxml.jackson.databind